test/fixtures/modifyVars.less

```diff
@@ -1,8 +1,9 @@
 @imgPath: '/path/to/images';
 @customPadding: 12px;
+@customPadding: undefined;
```

Comment:
@XhmikosR : I am not sure how to emulate a global variable with an undefined value (similar to how it could be passed in from the command line or from a grunt configuration).

What do you think? is it enough? The test passes of course.